### ISSUE TITLE
[a11y]: ai-chat should not only use placeholder text to convey instructions

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#879]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

The User instruction given with in the Chat Edit Field is presented in "Place holder" which is not a standard practice as per Accessibility standards , also the same is not descriptive.

Recommended fix: Placeholder is not a recommended method of conveying info as per accessibility standards , avoid the same :1. Provide a clear, descriptive, instruction as a visible label or helper text that sets the user’s expectation, e.g. “Type your queries related to IBM Cloud ".

Programmatically associate the instruction with the chat input field using appropriate Technics.

User impact: Impact Screen reader , Keyboard-only and cognitive users: Because the placeholder disappears on typing, they lose access to the instruction once they begin entering text, leading to uncertainty or errors.

<img width="1920" height="961" alt="Image" src="https://github.com/user-attachments/assets/8f8dd272-16d6-4523-b440-60ede788534e" />

### WCAG 2.1 Violation

WCAG Mapping: 3.3.2 Labels or Instruction

### Reproduction/example

n/a

### Steps to reproduce

1. make sure JAWS is active
2. open ai-chat
3. ask a question
4. see that the instruction "type something..." is given only as placeholder

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
